Northstar.CustomServers/mod/scripts/vscripts/evac/_evac.gnut

```diff
@@ -69,11 +69,23 @@ struct {
 	entity evacIcon
 } file
 
+// nessie fix
```

Comment:
Comments like this aren't really needed

Reply:
alright these are left for my own mod "tempfix", forgot to delete them